foam_controlwrapper/foam_controlwrapper.py

```diff
@@ -99,7 +99,7 @@ def run(self):
         # save timestep to mesh
         mesh._time = runner.get_last_time()
 
-    def add_mesh(self, mesh):
+    def add_dataset(self, mesh):
```

Comment:
I think we would want to check here to make sure that only ABCMesh (and not ABCLattice or ABCParticles) are added